docs/GOVERNANCE.md

```diff
@@ -0,0 +1,96 @@
+* Te gebruiken in WordPress
+* Commonground gedachte
+* Delen met anderen
+* Helpen met fouten
+* Het is een Datadienst of een Toepassing
+
+## Behoefte Bouwblokken
+Voor deze bouwblokkken kunnen we dan ook bepaalde eisen ophalen:
+* WordPress: opensource
+* Toegankelijk voor nieuwe nieuwe partijen die willen participeren
+* Pragmatische insteek, alleen formeel wanneer het nodig is
+* Ook een reeële manier om weer te verlaten
+* Garanties ten opzichte van elkaar
+* Bouwblokken hebben altijd een [publiccode.yaml](https://yml.publiccode.tools/)
+* Bouwblokken staan bij voorkeur op [github.com/openwebconcept](https://github.com/openwebconcept)
+* Losse componenten:
+    * Code deelbaar
+    * Zelfde licentie
+    * 2e-lijns support op de code (bugs/changes)
+    * Eén jaar aangeven dat support vervalt
+    * Eén plek waar alle bouwblokken te vinden zijn
+    * Signaal wanneer er een nieuwe versie is
+
+## Invulling Bouwblokken
+Per bouwblok:
+* Is deze te vinden op https://github.com/OpenWebconcept/
+* De licentie is EUPL
+* Er is één repository met die bloknaam
+* De gemeente is de beheerder van het bouwblok
+* De leverancier plaatst de code en maakt de aanpassingen. Een gemeente kan ook een leverancier zijn.
+* De beheerder kan een leverancier zijn of gemeenten maar MOET zijn opgenomen in de publicode.yaml
+* De eigenaar kan een gemeente zijn of leverancier maar MOET zijn opgenomen in de publicode.yaml
+* Issues en changes worden actief opgepakt
+* De beheerder van een bouwblok is verantwoordelijk voor de ondersteuning van andere partijen, ook als dit directe concurrenten zijn
+* Aanpassingen worden aangeleverd op basis van Pull Requests en bij voorkeur door een andere leverancier of gemeente goedgekeurd
+* Een bouwblok mag geen afhankelijkheden hebben op closed source software, tenzij expliciet vermeld in de readme
+
+### Techniek
+* Versiebeheer conform [semver](https://semver.org/)
+* Bij iedere nieuwe versie wordt er ook een github release aangemaakt
+* Bij iedere nieuwe github release wordt ook een package aangemaakt (zodat oudere versies later nog te downloaden en gebruiken zijn)
+* Het is installeerbaar via de WordPress plugin structuur (dus [WordPress update checker](https://github.com/YahnisElsts/plugin-update-checker) implementeren)
+* Naast plugin structuur moet ook installatie via [Composer](https://getcomposer.org/) worden ondersteund
+* Een bouwblok moet standaard [vertalingsopties](https://codex.wordpress.org/I18n_for_WordPress_Developers) ondersteunen
+* De blokken kunnen met elkaar draaien op dezelfde server, maar mogen ook los geïnstalleerd worden
+* Een bouwblok is voorzien van correcte en passende linters
+* Er is een workflow die bij PR's controleert of branches voldoen aan linters en andere voorwaarden
+* De algemene codestijl is de [WordPress coding standaard](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/). Een repository mag hiervan afwijken, maar moet dit dan documenteren
```

Comment:
Tijdens de meeting werd het advies nog gegeven om: https://wordpress.org/plugins/plugin-check/ te gebruiken om te controleren op veelvoorkomende fouten, dit is een plugin vanuit het WordPress Review Team is het handig om deze hier ook te zetten?

Reply:
Eens. En dan in de vorm van: "Een plugin zou alle tests van de [Plugin Check](https://wordpress.org/plugins/plugin-check/) moeten doorstaan"?

Weet jij of er een manier is om die checks in een pipeline te krijgen?